developer/js/libexec/create-override-script-regexp.ts

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env npx ts-node
```

Comment:
Maybe add a comment at the top of this file about what this script is for (corresponding to the comment in script-overrides-decorator.ts?